src/sass/_banner.scss

```diff
@@ -0,0 +1,30 @@
+// custom rules applied to gravity ui scss
+header[role=banner] {
+
+  .medium-logo {
+    fill: #ece9e9; // medium off-white
+  }
+
+  &.is-light {
+    .medium-logo {
+      fill: #000; // medium black
+    }
+  }
+
+  .grav-c-nav-menu__external-link {
+    .grav-c-block-link {
+      border-color: transparent;
+    }
+  }
+
+  .grav-c-block-link {
+    display: flex;
```

Comment:
These properties are only for the blog link, no? So, they should be within the `.grav-c-nav-menu__external-link { .grav-c-block-link { ... } }` block above.

Also, a (probably) unintended side-effect of this, is that on small screens all nav links have left-aligned text. instead of right-aligned:

![image](https://user-images.githubusercontent.com/16718916/38808725-16536dde-4179-11e8-8084-80c7566ed320.png)

Moving the CSS as described above will make the normal nav links revert back to the correct look. For the blog link itself, you'll need to add `justify-content: flex-end;` to fix it.